config/initializers/test_schema.rb

```diff
@@ -0,0 +1,5 @@
+if Rails.env.test?
+  unless Foreman.in_rake?('db:test:prepare')
```

Comment:
The `ActiveRecord::Migration.maintain_test_schema!` calls that rake task from [here](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/migration.rb#L583). Without that condition it would recurse infinitely